src/components/NewBook.jsx

```diff
@@ -1,22 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
- [ ] You are using `uuid` for your unique id generator, but you have no installed the package. Please do that.

  ![uuid](https://user-images.githubusercontent.com/54848363/185420088-d4f1ce41-d2c9-4d0a-a8a9-d4dd6ecc3766.jpg)